src/UI/sd-ui/src/components/teams/TeamCard.css

```
@@ -260,3 +260,18 @@
+
+/* Screen-reader-only text: present in the accessibility tree, off-screen
+   visually. Used to explain the record parenthetical, since a title on a
+   non-focusable element is not announced. */
+.visually-hidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)